lib/queue.rb

```diff
@@ -1,16 +1,17 @@
+require 'linked_list.rb'
+
 class Queue
```

Comment:
With the Queue you were supposed to use a circular buffer.  So this works, but it's not the assignment.